LayoutKitTests/StackViewTests.swift

```diff
@@ -97,6 +97,39 @@ class StackViewTests: XCTestCase {
         ])
     }
 
+    /**
+     This tests to make sure we can add subviews to the stack view, lay them out, then
+     remove them, then add them back again.
+     */
+    func testReusableStackView() {
```

Comment:
I think we can simplify this test further. Ideally we just want this to test removeArrangedSubviews.

```
func testRemoveArrangedSubviews() {
    let view1 = FixedSizeView(width: 7, height: 14)
    let view2 = FixedSizeView(width: 18, height: 18.5)
    let view3 = FixedSizeView(width: 33.5, height: 23)

    let stackView = StackView(axis: .vertical)

    stackView.addArrangedSubviews([view1, view2, view3])
    XCTAssertEqual(stackView.intrinsicContentSize, CGSize(width: 33.5, height: CGFloat(14 + 18.5 + 23)))
    XCTAssertEqual(stackView.subviews.count, 3)

    stackView.removeArrangedSubviews()
    XCTAssertEqual(stackView.intrinsicContentSize, CGSize.zero)
    XCTAssertEqual(stackView.subviews.count, 0)
}
```